execution/utility.sh

```diff
@@ -151,7 +155,9 @@ function fatalDirectory() {
 }
 
 function fatalMandatory() {
-  fatal "Mandatory arguments missing. Check usage via -h option."
+  local name="$1"; shift
+  local status="$1"; shift
+  fatal "Mandatory argument missing: ${name}. Check usage via -h option." && return $status
```

Comment:
Won't this just return from this function whereas the use requires returning out of the caller function.